src/simularium/VisGeometry.ts

```diff
@@ -30,6 +30,7 @@ import {
     WebGLRendererParameters,
     Mesh,
     Quaternion,
+    MOUSE,
```

Comment:
Strange, I can't find an entry for `Mouse` or `MOUSE` in the threejs docs!

Reply:
https://threejs.org/docs/?q=orbit#examples/en/controls/OrbitControls.keys

Reply:
https://threejs.org/docs/?q=orbit#examples/en/controls/OrbitControls.mouseButtons